### PR TITLE
[FIX] maintenance_plan: Remake the computation of next date

### PR DESCRIPTION
**Pull request by etobella**
Solves #161

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Some adjustments here https://github.com/tegin/maintenance/pull/1.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM now :+1:

File: maintenance_plan/models/maintenance_plan.py [hunk 1]
```diff
@@ -112,37 +112,41 @@ def _compute_maintenance_count(self):
     @api.depends(
         "interval",
         "interval_step",
```
Comment (LoisRForgeFlow) [inline maintenance_plan/models/maintenance_plan.py:114]: I think `start_maintenance_date` should also be in the depends.

**Review by mariadforgeflow — APPROVED** [review]

**Pull request merged**